aip/general/0193.md

```diff
@@ -346,14 +348,14 @@ actionable resolution, not as an alternative to them; for example.
 Details of this object are summarized in the following fields, field
 descriptions, and examples:
 
-- `description string`
-  - Describes what the link offers.
-  - *Example*: "Troubleshooting documentation for STOCKOUT errors"
-- `url string`
-  - The URL of the link. For error documentation this must follow the
++ `description string`
+  + Describes what the link offers.
+  + *Example*: "Troubleshooting documentation for STOCKOUT errors"
++ `url string`
+  + The URL of the link. For error documentation this must follow the
     standardized format listed in the following example.
-  - *Example*:
-    https://cloud.google.com/PRODUCT/docs/ERROR-REASON
+  + *Example*:
+      https://cloud.google.com/PRODUCT/docs/ERROR-REASON
```

Comment:
Nit: Why switch to `+`? We are pretty inconsistent with which markdown bullet we use, but I don't think `+` is used much.

Reply:
It's inconsistent in this file. I'll change them all to minus.

Reply:
Ah I looked beyond the diff to see the `+` usage.

Thanks for changing things.

Reply:
Done. All minus (`-`) now.